src/controllers/user.controller.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const userService = require('../services/user.service');
+
+const getAll = (req, res) => {
+  res.json(userService.readAll());
+};
+
+const get = (req, res) => {
+  const { id } = req.params;
+  const user = userService.read(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.status(200).json(user);
+};
+
+const create = (req, res) => {
+  const body = req.body;
```

Comment:
You can use object destructuring to access the properties of req.body. This helps make the code cleaner and easier to understand. For example: const { name } = req.body instead of const body = req.body